macosx/ReadMe.md

```diff
@@ -0,0 +1,658 @@
+---
```

Comment:
| :exclamation:  Instructions are at present not up-to-date. Update in progress. |
|--------------------------------------------------------------------------------|